projects/igniteui-angular/src/public_api.ts

```diff
@@ -100,8 +93,6 @@ export * from './lib/splitter/public_api';
 export * from './lib/stepper/public_api';
 export * from './lib/date-range-picker/public_api';
 export * from './lib/date-common/public_api';
-export * from './lib/grids/column-actions/column-actions-base.directive';
```

Comment:
@kdinev This one, for whatever reason is in use in the grid-column-actions sample and the public api barrel under /column-actions doesn't export it

Reply:
@damyanpetev I see why it is useful, however I don't see it documented anywhere and all of the abstract properties are hidden from the docs. There's a demo of how to create a custom one, but there's no docs in the public documentation. I'm adding it to the barrel exports in the column actions public_api, but I'm tagging @dkamburov to include an issue for API docs/sample and a topic.

Reply:
Oh, I agree, it's just the dev demos were failing to run like that. And just in case we want to drop it, better in a future major version.